__tests__/actionCreators/resources.test.js

```diff
@@ -88,7 +92,7 @@ describe('retrieveResource', () => {
       await store.dispatch(retrieveResource(currentUser, uri))
 
       const actions = store.getActions()
-      expect(actions.length).toEqual(5)
+      expect(actions.length).toEqual(6)
```

Comment:
would it be clearer to write `expect(actions).toEqual([{type: 'RETRIEVE_STARTED'}, ...])`?

Reply:
I only want to test the first 2, not all 6.

Reply:
but we test on lines 96, 97, 112, 113, 114, 115... Isn't that all 6?

Reply:
Yes, yes we do. Fixing.